mobile-bridge-api.json

```diff
@@ -51,6 +51,12 @@
         "type": "map",
         "documentation": "Additional configuration parameters.",
         "mandatory": false
+      },
+      {
```

Comment:
Should we call it `manualTracing` or just `tracing` ? Disabling it will not disable also the auto - tracing feature ?

Reply:
it won't disable auto-tracing feature (which we do in RN via headers injection), so the name of this field is for the usage of `DdTrace` only, the necessary statement will be added to the docs.

Reply:
Oh...yes, you are right because there we are not using the Tracing API to generated the `trace/span` id to be injected in the header.